capacity4more/behat/features/group_status.feature

```diff
@@ -0,0 +1,91 @@
+Feature: Group Status
+  Test group status field
+
+  @api
+  Scenario: Check requested group as admin
+    Given I am logged in as user "mariecurie"
+    When  I start editing group "Requested group"
+    Then  the "#edit-c4m-og-status-und" element should not contain "archived"
+    And   the "#edit-c4m-og-status-und" element should not contain "published"
+
+  @api
+  Scenario: Check requested group as group owner
+    Given I am logged in as user "alfrednobel"
+    When  I start editing group "Requested group"
+    Then  the "#edit-c4m-og-status-und" element should not contain "rejected"
+    And   the "#edit-c4m-og-status-und" element should not contain "draft"
+    And   the "#edit-c4m-og-status-und" element should not contain "published"
+    And   the "#edit-c4m-og-status-und" element should not contain "archived"
+
+  @api
+  Scenario: Check rejected group group as admin
+    Given I am logged in as user "mariecurie"
+    When  I start editing group "Rejected group"
+    Then  the "#edit-c4m-og-status-und" element should not contain "requested"
+    And   the "#edit-c4m-og-status-und" element should not contain "draft"
+    And   the "#edit-c4m-og-status-und" element should not contain "published"
+    And   the "#edit-c4m-og-status-und" element should not contain "archived"
+
+  @api
+  Scenario: Check rejected group group as group owner
+    Given I am logged in as user "alfrednobel"
```

Comment:
@zero2one mariecure is not a group owner - it's site admin. Group owner is alfrednobel and he doesn't see the field so he can't change it https://github.com/capacity4dev/capacity4more/pull/537/files#diff-d14bf0915f4fa490a2acd63d3184bbf2R31